__tests__/app.test.js

```diff
@@ -58,18 +71,58 @@ describe("GET /api/topics", () => {
             .expect(200)
             .then((response) => {
                 const topics = response.body.topics
+                expect(topics.length).toBe(3)
                 topics.forEach((topic) => {
                     expect(topic).toHaveProperty('slug')
                     expect(topic).toHaveProperty('description')
                 })
             })
     })
-    it("returns a 404 status if summoned with an incorrect url", () => {
+})
+
+
+describe("GET /api/articles/:article_id", () => {
+    it("returns an article object", () => {
         return request(app)
-            .get('/api/not-topics')
-            .expect(404)
+            .get('/api/articles/1')
+            .expect(200)
             .then((response) => {
-                expect(response.res.statusMessage).toBe('Not Found')
+                const articleObj = response.body
+                expect(articleObj === Object(articleObj)).toEqual(true)
+            })
+    })
+    it("Object contains correct properties", () => {
+        return request(app)
+            .get('/api/articles/1')
+            .expect(200)
+            .then((response) => {
+                const article = response.body.article
+                expect(article).toHaveProperty('author')
+                expect(article).toHaveProperty('title')
+                expect(article).toHaveProperty('article_id')
+                expect(article).toHaveProperty('body')
+                expect(article).toHaveProperty('topic')
+                expect(article).toHaveProperty('created_at')
+                expect(article).toHaveProperty('votes')
+                expect(article).toHaveProperty('article_img_url')
             })
     })
-})
\ No newline at end of file
+    it("returns an object with only one article", () => {
+        return request(app)
+            .get('/api/articles/1')
+            .expect(200)
+            .then((response) => {
+                const articleArr = Object.values(response.body)
+                expect(articleArr.length).toBe(1)
+            })
+    })
+    it("returns a 400 status if no article is found with the given ID", () => {
+        return request(app)
+            .get('/api/articles/99999999')
+            .expect(400)
```

Comment:
This would be better as a 404 not found rather than a 400 bad request. The request is valid but that article just cannot be found.